pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -452,12 +452,16 @@ func makePlan(
 		// spans that are assigned to it.
 		// We could compute per-aggregator checkpoint, but that's probably an overkill.
 		var aggregatorCheckpoint execinfrapb.ChangeAggregatorSpec_Checkpoint
+		//lint:ignore SA1019 deprecated usage
+		var changeFrontierCheckpoint jobspb.ChangefeedProgress_Checkpoint
```

Comment:
This extra variable seems unnecessary, you can just pass the checkpoint directly